### PR TITLE
[Non modular] Small booze changes to make it more sensible

### PR DESCRIPTION
**Pull request by Azarak**
## About The Pull Request
Now you get drunk 60% slower, but your drunkiness decays around 300% slower. Allowing people to get smashed after a proper amount of alcohol, not just one glass, but also making so the alcohol persists through a long duration of time.
This is the change we've had before and before

## Why It's Good For The Game
more RP friendly drunkiness

## Changelog
:cl:
/:cl:

### CODE REVIEW

**Comment by Gandalf2k15** [comment]
Perhaps an upstream change?

**Comment by Azarak** [comment]
> 
> 
> Perhaps an upstream change?

TG rounds end up being quite short most often than not, not sure if this is something they'd like

File: code/modules/mob/living/carbon/life.dm [hunk 1]
```diff
@@ -505,7 +505,7 @@ All effects don't start immediately, but rather get worse over time; the rate is
 		handle_hallucinations()
 
 	if(drunkenness)
-		drunkenness = max(drunkenness - (drunkenness * 0.04) - 0.01, 0)
```
Comment (Gandalf2k15) [inline code/modules/mob/living/carbon/life.dm:508]: Remember to have original

**Review by Gandalf2k15 — CHANGES_REQUESTED** [review]

File: code/modules/mob/living/carbon/life.dm (hunk 1 above)
Comment (Azarak) [reply code/modules/mob/living/carbon/life.dm:508]: done!

**Pull request merged**